gax-java/gax/src/main/java/com/google/api/gax/rpc/EndpointContext.java

```diff
@@ -117,6 +124,9 @@ public static EndpointContext getDefaultInstance() {
   @Nullable
   public abstract MtlsProvider mtlsProvider();
 
+  @Nullable
+  public abstract CertificateBasedAccess certificateBasedAccess();
```

Comment:
I believe the others getters are public, but they probably shouldn't have been.

Since this is new, can we make this package-private?